packages/api/src/utils/logs.js

```diff
@@ -268,9 +298,25 @@ export class Logging {
     if (this.opts?.debug) {
       console.log(`[${this._date()}] `, `${name}: ${duration} ms`)
     }
-    return timeObj
+
+    if (shouldLog) {
```

Comment:
Note that timing execution is not effective in production and I believe that is why this method does not send logs to logtail.

https://developers.cloudflare.com/workers/learning/security-model/#step-1-disallow-timers-and-multi-threading